conversation_reconstruction/dataflow/conversation_reconstruction/dataflow_reingested.py

```diff
@@ -60,31 +58,29 @@ def run(known_args, pipeline_args):
 
-  avro_mapping = lambda x: (x['page_id'], x)
-  json_mapping_tmp = lambda x: (x[0]["page_id"], x[0]) if (type(x[0]) is dict) else ((ast.literal_eval(x)["page_id"], ast.literal_eval(x)) if (type(ast.literal_eval(x)) is dict) else (ast.literal_eval(x)[0]["page_id"], ast.literal_eval(x)[0]))
   json_mapping = lambda x: (json.loads(x)["page_id"], json.loads(x))
   with beam.Pipeline(options=pipeline_options) as p:
-   # Read from BigQuery
+    # Read from BigQuery
     to_be_processed = (p | 'Read_to_be_processed' >> beam.io.ReadFromText(known_args.input.format(week=known_args.week, year=known_args.year))
                           | 'INGESTED_assign_page_id_as_key' >> beam.Map(json_mapping))
-    last_revision = (p | 'Retrieve_last_revision' >> beam.io.ReadFromText("gs://wikidetox-viz-dataflow/process_tmp/%s/*last_rev*"%known_args.category)| 'LASTREV_assign_page_id_as_key' >> beam.Map(json_mapping))
-    page_state = (p | 'Retrieve_page_state' >> beam.io.ReadFromText("gs://wikidetox-viz-dataflow/process_tmp/%s/*page_states*"%known_args.category)| 'PAGESTATE_assign_page_id_as_key' >> beam.Map(json_mapping))
-
-    reconstruction_results, page_states, input_last_rev, last_rev_output, input_page_states\
+    last_revision_location = "gs://wikidetox-viz-dataflow/process_tmp/%s/*last_rev*"
+    last_revision = (p | 'Retrieve_last_revision' >> beam.io.ReadFromText(last_revision_location%known_args.category)
+                       | 'LASTREV_assign_page_id_as_key' >> beam.Map(json_mapping))
+    page_state_location = "gs://wikidetox-viz-dataflow/process_tmp/%s/*page_states*"
+    page_state = (p | 'Retrieve_page_state' >> beam.io.ReadFromText(page_state_location%known_args.category)
+                    | 'PAGESTATE_assign_page_id_as_key' >> beam.Map(json_mapping))
+    reconstruction_results, page_states, last_rev_output\
                    = ({'to_be_processed': to_be_processed, 'last_revision': last_revision, 'page_state': page_state}
-                   | beam.CoGroupByKey()
                    # Join information based on page_id
-                   | beam.ParDo(ReconstructConversation()).with_outputs('page_states', 'last_revision', 'last_revision_output', 'input_page_state', main = 'reconstruction_results'))
+                   | beam.CoGroupByKey()
+                   | beam.ParDo(ReconstructConversation()).with_outputs('page_states','last_revision_output', main = 'reconstruction_results'))
                    # Reconstruct the conversations
- 
-    page_states | "WritePageStates" >> beam.io.WriteToText("gs://wikidetox-viz-dataflow/process_tmp/next_%s/page_states"%known_args.category) 
-    # Saving page states and last processed revision to BigQuery, and the last time last processed revision to cloud for bak up
-    reconstruction_results | "WriteReconstructedResults" >> beam.io.WriteToText("gs://wikidetox-viz-dataflow/reconstructed_res/%s"%jobname) 
+    # Saving page states and latest processed revisionn to cloud for future
```

Comment:
typo revisionn
also, future processing (?)
(fyi at google we punctuate comments. might be a good habit.)